cfpb/hmda-frontend
# [Dev] Fix 'process is not defined' error when saving changes

Closes #1586 

### Testing
- yarn install
- Launch dev server
- Open http://localhost:3000
- Open dev tools -> console
- Open src > homepage > index.js and change `<h1>The Home Mortgage Disclosure Act</h1>` to `<h1>Change</h1>`
- Save changes
- Notice that the `process is not defined` error does not appear in the console
- Notice that the page remain interactive (can click links)
